content/product/cloud-delivery-service.md

```diff
@@ -0,0 +1,15 @@
+---
+title: "Cloud Delivery Service"
+date: 2018-06-01T15:10:06-07:00
```

Comment:
As a subsequent exercise, we should consider deleting the dates from most of these content files, as I think they're just "dead code", except for blog posts where date _is_ used.